build.gradle

```diff
@@ -4019,6 +4020,7 @@ project(":web") {
                     } else {
                         targetCpuBitDepthSwitch = "--32-bit"
                     }
+                    cmakeArgs += " -DWEBVIEW_RELEASE_VERSION=${WEBVIEW_VERSION}"
```

Comment:
Since this is a browser version, as noted in the PR Description, I recommend changing this variable name here and in the below files to `WEBVIEW_BROWSER_VERSION`.